packages/aws-cdk/lib/api/aws-auth/sdk-provider.ts

```diff
@@ -383,7 +383,7 @@ function parseHttpOptions(options: SdkHttpOptions) {
     // https://aws.amazon.com/blogs/developer/using-the-aws-sdk-for-javascript-from-behind-a-proxy/
     debug('Using proxy server: %s', proxyAddress);
     // eslint-disable-next-line @typescript-eslint/no-require-imports
-    const ProxyAgent: any = require('proxy-agent');
+    const ProxyAgent: any = require('../../../proxy-agent');
```

Comment:
Would recommend making it such that it's:

```suggestion
    const ProxyAgent: any = require('../../../vendor/proxy-agent');
```